libraries/adaptive-expressions/adaptive/expressions/expression.py

```diff
@@ -242,3 +242,47 @@ def try_evaluate(self, state: MemoryInterface, options: Options = None):
         return self.evaluator.try_evaluate(self, state, options)
 
     # TODO: toString
+    def to_string(self):
+        import constant as const
+        builder = ""
+        valid = False
+        expr_type = self.evaluator.expr_type
```

Comment:
>expr_type = self.evaluator.expr_type [](start = 8, length = 36)

this can be self.expr_type, no need to new this expr_type

Reply:
fixed